charts/kyverno/templates/deployment.yaml

```diff
@@ -30,9 +30,15 @@ spec:
       {{- with .Values.podSecurityContext }}
       securityContext: {{ tpl (toYaml .) $ | nindent 8 }}
       {{- end }}
-      {{- if and .Values.antiAffinity.enable .Values.affinity }}
+      {{- if .Values.affinity.enable }}
```

Comment:
Based on my other comment about simplify, could do this here:

```
{{- if or .Values.affinity.podAntiAffinity.enable .Values.affinity.otherAffinity.enable }}
```

Reply:
done